token/services/db/dbtest/notifier.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
```

Comment:
Moved all test cases under the same detest package. I don't expose them directly, but through the test method that is called by each driver: memory, sqlite, postgres